### PR TITLE
Remove compatibility flag Set Bracketing Last Introduction Pattern

### PR DESCRIPTION
**Pull request by herbelin**
**Kind:**  feature

This was a compatibility flag since 8.5, set to true by default since version 8.6, deprecated since 8.10.

- [X] Updated test-suite
- [X] Corresponding documentation was updated
- [x] Entry added in the changelog

### CODE REVIEW

**Comment by herbelin** [comment]
Only incompatilities are in fiat-parsers.

File: doc/changelog/04-tactics/13509-master+remove-bracketing-last-introduction-pattern-flag.rst
Comment (Zimmi48) [inline doc/changelog/04-tactics/13509-master+remove-bracketing-last-introduction-pattern-flag.rst:2]:
```suggestion
  Deprecated flag :flag:`Bracketing Last Introduction Pattern` affecting the
```

**Review by Zimmi48 — APPROVED** [review]

**Comment by herbelin** [comment]
Note: there is a backward-compatible fix for fiat-parsers at mit-plv/fiat#48. Waiting for it to be merged, then, no overlay will be needed.

**Comment by SkySkimmer** [comment]
That counts as needs: overlay until it's merged imo

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
Let's wait for the overlay to be merged to run a clean CI.

**Comment by ppedrot** [comment]
Overlay merged upstream, @herbelin can you rebase?

**Pull request merged**
